src/services/Forum/HandleForumAdmin.js

```diff
@@ -0,0 +1,98 @@
+import { firebase } from "../Firebase/Config";
+
+export async function addAdmin(forumId, adminDetails, callbackSuccess) {
+  await firebase
+    .firestore()
+    .collection("forums")
+    .doc(forumId)
+    .collection("admins")
+    .doc(adminDetails.uid)
+    .set(adminDetails)
+    .then(() => callbackSuccess());
```

Comment:
It would be better if the user receives notification when he/she gets promoted to admin, or vice versa.